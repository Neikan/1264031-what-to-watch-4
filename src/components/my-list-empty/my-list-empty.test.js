// Импорт библиотек
import React from "react";
import renderer from "react-test-renderer";

// Импорт компонентов
import MyListEmpty from "./my-list-empty.jsx";


describe(`Test MyListEmpty component`, () => {
  test(`MyListEmpty component is created and rendered correctly`, () => {
    const tree = renderer.create(
        <MyListEmpty />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
