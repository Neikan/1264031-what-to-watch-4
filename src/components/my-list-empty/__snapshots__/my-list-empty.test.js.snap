// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MyListEmpty component MyListEmpty component is created and rendered correctly 1`] = `
Array [
  <p>
    Your favorites list is empty...
  </p>,
  <p>
    Please add your first movie!
  </p>,
]
`;
